kaldb/pom.xml

```diff
@@ -166,21 +168,15 @@
         <dependency>
             <groupId>org.opensearch</groupId>
             <artifactId>opensearch</artifactId>
-            <version>2.3.0</version>
+            <version>${opensearch.version}</version>
         </dependency>
         <dependency>
             <groupId>org.opensearch.plugin</groupId>
             <artifactId>lang-painless</artifactId>
-            <version>2.3.0</version>
+            <version>${opensearch.version}</version>
         </dependency>
 
         <!-- Kafka writer dependencies -->
-        <dependency>
```

Comment:
given we might consider moving away from kafka streams maybe we can keep this as an explicit dependency?

Reply:
Done.